lib/sub-process.ts

```diff
@@ -1,10 +1,16 @@
 import * as childProcess from 'child_process';
 
-export function execute(
+export { execute, Output };
+interface Output {
```

Comment:
Consider renaming to something for explicit, like `CmdOutput` - as in other places where you do:
```js
import {Output} from './sub-process'
```
the `Output` type can be confusing as it's a very generic name